Cargo.toml

```diff
@@ -30,6 +30,18 @@ members = [
 
 [dev-dependencies]
 no-panic = "0.1.8"
+rand = "0.6.5"
+paste = "0.1.5"
 
 [build-dependencies]
 rand = { version = "0.6.5", optional = true }
+
+[profile.bench]
+opt-level = 3
```

Comment:
Oh one thing I Forgot to mention is that this section can likely be almost entirely removed. The only setting which is set to something that's not the default is `lto = true`, but that probably isn't too useful for these benchmarks since it may not represent what callers will be benchmarking. (you typically don't LTO with libm)

Reply:
done